fastlane/Fastfile

```diff
@@ -86,6 +86,22 @@ platform :android do
       setGradleOutputsInEnv()
     end
 
+  desc "Generate AdHoc HybridApp apk"
+    lane :build_adhoc_hybrid do
+      ENV["ENVFILE"]="../.env.adhoc.hybridapp"
+      gradle(
+        project_dir: '../Android',
+        task: 'assembleAdhoc',
+        properties: {
+            "android.injected.signing.store.file" => './upload-key.keystore',
+            "android.injected.signing.store.password" => ENV["ANDROID_UPLOAD_KEYSTORE_PASSWORD"],
+            "android.injected.signing.key.alias" => ENV["ANDROID_UPLOAD_KEYSTORE_ALIAS"],
+            "android.injected.signing.key.password" => ENV["ANDROID_UPLOAD_KEY_PASSWORD"],
```

Comment:
Do we use the same signing key for HybridApp builds on `App`? I thought they might be different

Reply:
I believe so because I copied part of the workflow from `deploy.yml` where we use that signing key:
<img width="1103" alt="image" src="https://github.com/user-attachments/assets/7d6c1256-8c1f-4039-bd8d-979199492d8f">